### PR TITLE
Core release v39.0.0 + va-memorable-date: remove non-USWDS version

### PR DESCRIPTION
**Pull request by jamigibbs**
## Chromatic

https://2660-mem-date-v1-remove--65a6e2ed2314f7b8f98609d8.chromatic.com

## Description
This will remove the non-USWDS version of `va-memorable-date`. Only the USWDS variation will be available now.

Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2660

## QA Checklist
- [x] Component maintains 1:1 parity with design mocks
- [x] Text is consistent with what's been provided in the mocks
- [x] Component behaves as expected across breakpoints
- [ ] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [ ] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [ ] Tab order and focus state work as expected
- [ ] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [ ] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [ ] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Acceptance criteria
- [ ] QA checklist has been completed
- [ ] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-memorable-date/va-memorable-date.scss [hunk 1]
```diff
@@ -13,103 +13,58 @@
...
-    flex: 0 1 50%;
-    width: 50%;
-  }
-}
+@import '~@department-of-veterans-affairs/css-library/dist/elements.css';
 
-:host(:not([uswds='false'])) {
+:host {
+  display: block;
+  font-family: var(--font-source-sans);
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-memorable-date/va-memorable-date.scss:22]: The `va-date` stylesheet was being imported previously for the v1 variation but there was still a few styles that the USWDS version was relying on. We did not need to the entire stylesheet though.

So I have removed that import `@import '../va-date/va-date.css';` and instead added the relevant styles from there like these here.

**Comment by jamigibbs** [comment]
We are going to have a lot of conflicts with our [Formation base font size PR](https://github.com/department-of-veterans-affairs/component-library/pull/1096) as we do these non-USWDS deprecations so after this is merged, I will make sure to go and resolve those conflicts there.

**Comment by powellkerry** [comment]
@jamigibbs do we know if there are any v1 versions still in use on vets-website?

**Review by powellkerry — APPROVED** [review]

**Comment by jamigibbs** [comment]
> @jamigibbs do we know if there are any v1 versions still in use on vets-website?

Great question. I could not find any instances and confirmed in the component usage report too: https://docs.google.com/spreadsheets/d/1IKi_kNSQtasTb3hHbcKbhHLv7BCYCmRqsUtaRPcrtG8/edit#gid=299423802

**Review by harshil1793 — APPROVED** [review]

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**